Makefile.am

```diff
@@ -4,9 +4,8 @@ SUBDIRS = data po src tests doc apidoc
 EXTRA_DIST = \
     libreport.pc.in \
     libreport-version \
-    asciidoc.conf
-
-dist_doc_DATA = README.md
+    asciidoc.conf \
+	README.md
```

Comment:
Tab here should be 4 spaces to maintain consistency.

Reply:
Thanks, fixed.